in_text/text_01_spatial.yml

```diff
@@ -153,155 +160,553 @@ entities:
     -
-      attr-label: n_obs_wu
+      attr-label: n_obs_su
+      attr-def: >-
+        The number of temperature observations in the summer used to categorize source lakes (not used as a metafeature for target lakes, 
+        even while appears in the data release)
+      attr-defs: This data release
+      data-min: 3
+      data-max: 30305
+      data-units: NA
     -
       attr-label: n_obs_au
+      attr-def: >-
+        The number of temperature observations in the autum used to categorize source lakes (not used as a metafeature for target lakes, 
+        even while appears in the data release)
+      attr-defs: This data release
+      data-min: 0
+      data-max: 13971
+      data-units: NA
     -
       attr-label: obs_depth_mean_frac
+      attr-def: >-
+        The mean of depth observations as a fraction of the maximum reported depth for the lake. This number exceeds 1 
+        in the case where the mean depth of observations was greater than the reported maximum depth of the lake, in those cases, the observations
+        below the maximum depth were excluded. This happened in some cases because a deeper basin was considered part of the same lake for 
+        temperature monitoring purposes, but considered a separate lake for reporting maximum depth, or is indicative of errors. A value of 
+        0 for a given lake means all measurements were taken at the surface. This value was not used as a metafeature for target lakes, 
+        even while appears in the data release.
+      attr-defs: This data release
+      data-min: 0
+      data-max: 2.12
+      data-units: NA
     -
       attr-label: obs_temp_mean
+      attr-def: >-
+        The mean temperature of observations. This value was not used as a metafeature for target lakes, 
+        even while appears in the data release.
+      attr-defs: This data release
+      data-min: 3.60
+      data-max: 25.3
+      data-units: degrees C
     -
       attr-label: obs_temp_std
+      attr-def: >-
+        The standard deviation of temperature of observations. This value was not used as a metafeature for target lakes, 
+        even while appears in the data release.
+      attr-defs: This data release
+      data-min: 1.44
+      data-max: 10.2
+      data-units: degrees C
     -
       attr-label: obs_temp_skew
+      attr-def: >-
+        The skew of temperature of observations. This value was not used as a metafeature for target lakes, 
+        even while appears in the data release.
+      attr-defs: This data release
+      data-min: NA
+      data-max: NA
+      data-units: NA
     -
       attr-label: obs_temp_kurt
+      attr-def: >-
+        The kurtosis of temperature of observations. This value was not used as a metafeature for target lakes, 
+        even while appears in the data release.
+      attr-defs: This data release
+      data-min: NA
+      data-max: NA
+      data-units: NA
     -
       attr-label: zero_temp_doy
+      attr-def: >-
```

Comment:
@jdwillard19 please check this for accuracy. You can comment here and I can update the PR with any changes.